llvm/lib/CodeGen/MachineLateInstrsCleanup.cpp

```diff
@@ -187,9 +187,26 @@ static bool isCandidate(const MachineInstr *MI, Register &DefedReg,
     const MachineOperand &MO = MI->getOperand(i);
     if (MO.isReg()) {
       if (MO.isDef()) {
+        // To get the \DefedReg value, we need to check that 1st MachineOperand
```

Comment:
This isnt' a /// comment, so the doxygen \ syntax won't do anything; plus I don't think it does anything for comments in the body of a function, only on the declaration